embark.el

```diff
@@ -230,19 +230,25 @@ Used by `embark-completing-read-prompter' and `embark-keymap-help'.")
 (defface embark-target '((t :inherit highlight))
   "Face used to highlight the target at point during `embark-act'.")
 
-(defcustom embark-indicator #'embark-mixed-indicator
-  "Indicator function to use when acting or becoming.
-The indicator function is called from both `embark-act' and from
-`embark-become' and should display information about this to the
-user, such as: which of those two commands is running; a
+(make-obsolete embark-indicator
+               "see the new `embark-indicators' variable."
+               "0.12")
```

Comment:
Does this make sense? Did you tag a release in the meantime? We went from embark-action-indicator -> embark-indicator -> embark-indicators.

Reply:
Oh, probably not! I'm supposed to tag 0.12 first, and then mention it in these messages, right? Not the other way around.

Reply:
I would not make a new release before this. But I usually only add deprecations based on the latest tagged release. Everything which happens in between is development and things can move around more freely.